packages/react-app/src/NftyWallet.js

```diff
@@ -95,427 +93,193 @@ export default function NftyWallet(props) {
-              {...props}
-              key={renderKey}
-              canvasKey={canvasKey}
-              address={props.address}
+              setInjectedProvider={props.setInjectedProvider}
+              setInjectedGsnSigner={setInjectedGsnSigner}
               mainnetProvider={props.mainnetProvider}
-              injectedProvider={props.injectedProvider}
-              metaProvider={props.metaProvider}
-              kovanProvider={props.kovanProvider}
-              readKovanContracts={props.readKovanContracts}
-              mode={mode}
-              ink={ink}
-              ipfsHash={ipfsHash}
-              setMode={setMode}
-              setIpfsHash={setIpfsHash}
-              setInk={setInk}
-              drawing={drawing}
-              setDrawing={setDrawing}
-              viewDrawing={viewDrawing}
-              setViewDrawing={setViewDrawing}
-              ipfsConfig={ipfsConfig}
-              ipfsConfigInfura={ipfsConfigInfura}
-              gasPrice={props.gasPrice}
-              calculatedVmin={calculatedVmin}
-              transactionConfig={transactionConfig}
+              price={props.price}
+              minimized={props.minimized}
+              setMetaProvider={props.setMetaProvider}
             />
           </div>
-        </Route>
 
-        <Route path="/ink/:hash">
-          <div>
-            <ViewInk
-              {...props}
-              address={props.address}
-              artist={artist}
-              calculatedVmin={calculatedVmin}
-              canvasKey={canvasKey}
-              drawing={drawing}
-              gasPrice={props.gasPrice}
-              injectedProvider={props.injectedProvider}
-              ink={ink}
-              ipfsConfig={ipfsConfig}
-              ipfsConfigInfura={ipfsConfigInfura}
-              ipfsHash={ipfsHash}
-              key={renderKey}
-              kovanProvider={props.kovanProvider}
-              mainnetProvider={props.mainnetProvider}
-              metaProvider={props.metaProvider}
-              readContracts={props.readContracts}
-              readKovanContracts={props.readKovanContracts}
-              setArtist={setArtist}
-              setDrawing={setDrawing}
-              setInk={setInk}
-              setIpfsHash={setIpfsHash}
-              setMode={setMode}
-              setTab={setTab}
-              setViewDrawing={setViewDrawing}
-              transactionConfig={transactionConfig}
-              upgradePrice={upgradePrice}
-              viewDrawing={viewDrawing}
+          <Switch>
+            <Route exact path="/">
+              <Homepage
+                localProvider={props.localProvider}
+                injectedProvider={props.injectedProvider}
+              />
+            </Route>
+            <Route path="/debug">
+              <DebugContracts {...props} />
+            </Route>
+            <Route path="/recent">
+              <AllInks />
+            </Route>
+            {/* <Route path="/curated">
```

Comment:
We actually want to have this page in production.

Reply:
UPD: I see that CuratedInks is used on the HomePage. My apologies.

Reply:
@ironsoul0 should we remove the fake data after merge then ?